src/UdpDataProtocol.cpp

```diff
@@ -808,7 +808,12 @@ void UdpDataProtocol::receivePacketRedundancy(
             }
             src = dst;
         }
-        if (!mJackTrip->writeAudioBuffer(src, host_buf_size, gap_size)) {
+        int ok = true; // send audio buf to
+        ok = (mJackTrip->getBufferStrategy() !=3) ? // ring or jitter
```

Comment:
This can be one.

Reply:
that differs in the last arg -- gap_size in the original case, last_seq_num in the new case

Reply:
I've meant `int ok = (mJackTrip…`. `true` in line 811 will be immediatley overridden. And it looks like it would be bool. Should it be bool?